### DISCUSSION TITLE
Whats  the reason to only allow non partitioned inputs in the fully featured example?

### DISCUSSION THREAD

**Discussion by pwachira** [dagster-io/dagster#9263] [Q&A]
https://github.com/dagster-io/dagster/blob/e43a821334354b8f9002868578f62d0a348c9da6/examples/project_fully_featured/project_fully_featured/resources/duckdb_parquet_io_manager.py#L34

**Comment by sryza** [comment]
This particular pipeline didn't have any places where the duckdb parquet IO manager needed to load partitioned inputs, so we didn't implement it, but it would be pretty straightforward to get working.

**Comment by sryza** [comment]
Also relevant: https://github.com/dagster-io/dagster/pull/9375
